sample-apps/bundles-cart-transform/extensions/cart-expand-js/shopify.extension.toml

```
@@ -1,8 +1,15 @@
+api_version = "2024-04"
+
+[[extensions]]
 name = "test-demo"
 type = "cart_transform"
 title = "cart transform"
```

Just reading the public docs [here](https://shopify.dev/docs/apps/build/app-extensions/configure-app-extensions#extension-types), I think this should be `type = "function"` (unless this a recent/undocumented change). Also, it looks like `title` doesn't exist as a valid property (should just be `name`).

This would apply to both toml files.

```suggestion
type = "function"
```

---

You are right, updated.